VitalClient/src/main/java/io/tryvital/client/Configuration.kt

```diff
@@ -1,7 +1,40 @@
 package io.tryvital.client
 
-data class Configuration(
-    val apiKey: String,
-    val region: Region,
-    val environment: Environment
-)
+import android.content.SharedPreferences
+
+const val VITAL_ENCRYPTED_USER_ID_KEY: String = "userId"
+const val VITAL_ENCRYPTED_REGION_KEY = "region"
+const val VITAL_ENCRYPTED_ENVIRONMENT_KEY = "environment"
+const val VITAL_ENCRYPTED_API_KEY_KEY = "apiKey"
+
+internal interface ConfigurationReader {
+    val userId: String?
+    val region: Region?
+    val environment: Environment?
+    val apiKey: String?
+}
```

Comment:
Why do we need an interface for this?

Reply:
Testable by dependency injection

The main implementation reads from the `SharedPreferences` (= UserDefaults)
The test implementation has it as stored property.

Both implementations are in this same file